### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ctrongminh**
Update the README.md as per https://github.com/parse-community/parse-dashboard/issues/798

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -317,6 +317,7 @@ var dashboard = new ParseDashboard({
 
 ### Makings users read-only
 
+Make sure you specify the `readOnlyMasterKey` for the apps that you want to use read-only feature in "apps" configuration.
```
Comment (flovilmart) [inline README.md]: Can we have an example here of readOnlyMasterKey in the apps options?

Comment (ctrongminh) [reply README.md]: @flovilmart sure.

Making users readonly. I have not tried to use parse dashboard with two apps but I think the below config file should work fine.
```json
{
  "apps": [
    {
      "appId": "myAppId1",
      "masterKey": "myMasterKey1",
      "readOnlyMasterKey": "myReadOnlyMasterKey1",
      "serverURL": "myURL1",      
      "port": 4040,
      "production": true
    },
    {
      "appId": "myAppId2",
      "masterKey": "myMasterKey2",
      "readOnlyMasterKey": "myReadOnlyMasterKey2",
      "serverURL": "myURL2",      
      "port": 4041,
      "production": true
    }
  ],
  "users": [
    {
      "user":"user1",
      "pass":"pass1",
      "readOnly": true,
      "apps": [{"appId": "myAppId1"}, {"appId": "myAppId2"}]
    },
    {
      "user":"user2",
      "pass":"pass2",
      "apps": [{"appId": "myAppId1"}]
    }
  ]
}
```

Making users' apps readonly
```json
{
  "apps": [
    {
      "appId": "myAppId1",
      "masterKey": "myMasterKey1",
      "readOnlyMasterKey": "myReadOnlyMasterKey1",
      "serverURL": "myURL",      
      "port": 4040,
      "production": true
    },
    {"...": "..."}
  ],
  "users": [
    {
      "user":"user",
      "pass":"pass",
      "apps": [{"appId": "myAppId", "readOnly": true}, {"appId": "myAppId2"}]
    }
  ]
}
```

Comment (flovilmart) [reply README.md]: yes that looks OK!

Comment (ctrongminh) [reply README.md]: I committed the changes

**Pull request merged**